storage/gcloud/aio/storage/storage.py

```diff
@@ -504,7 +504,7 @@ async def _upload_multipart(self, url: str, object_name: str,
 
         parts = [
             (metadata_headers, json.dumps(metadata).encode('utf-8')),
-            ({'Content-Type': headers['Content-Type']}, bytes_body),
+            ({'content-type': headers['Content-Type']}, bytes_body),
```

Comment:
Is the casing change here necessary, or just a dev artifact? I believe we're using Title-Cased headers throughout this lib, so if Google is doing weird things with them, I might need to look into updating this more generally.

Reply:
In fact the case is not necessary.

Reply:
I just pushed a new commit where I "stringnify" every metadata.metadata fields. This seems necessary if there are list or dict fields (for simple types the conversion was automatic).